src/ResourceManager/Compute/Commands.Compute/help/Get-AzureRmGalleryImageDefinition.md

```diff
@@ -0,0 +1,129 @@
+---
+external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+Module Name: AzureRM.Compute
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/get-azurermgalleryimagedefinition
+schema: 2.0.0
+---
+
+# Get-AzureRmGalleryImageDefinition
+
+## SYNOPSIS
+Get or list gallery image definitions.
+
+## SYNTAX
+
+### DefaultParameter (Default)
+```
+Get-AzureRmGalleryImageDefinition [-ResourceGroupName] <String> [-GalleryName] <String>
+ [[-GalleryImageDefinitionName] <String>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameter
+```
+Get-AzureRmGalleryImageDefinition [-ResourceId] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get or list gallery image definitions.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzureRmGalleryImageDefinition -ResourceGroupName $rgname -GalleryName $gallery -GalleryImageDefinitionName $image
+```
+
+Get the gallery image definition.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -GalleryImageDefinitionName
+The name of the gallery image definition
+
+```yaml
+Type: System.String
+Parameter Sets: DefaultParameter
+Aliases: Name
```

Comment:
@hyonholee in this case, it's good to have `-GalleryName` stay as it is, but swap the name and alias of this parameter